sirepo/pkcli/job_supervisor.py

```diff
@@ -158,11 +159,22 @@ def post(self):
 class _ServerReqRunMulti(_ServerReq):
 
     async def post(self):
+        async def _await_reply(content, handler):
+            r = copy.deepcopy(content.data)
+            del r['models']
```

Comment:
Doesn't this delete models.simulation? I don't know if that's used.

Reply:
Yes, it deletes all models. Models aren't useful for the GUI to know which request the response is to. All the GUI needs is the report (scan uuid). Really that is a problem because the server uses computeJid/hash/serial to identify which are things the GUI doesn't know much about....